lib/matplotlib/table.py

```diff
@@ -769,6 +803,22 @@ def table(ax,
     else:
         cellColours = ['w' * cols] * rows
 
+    if edgeColour is None:
+        # default edge color is black
+        edgeColour = 'k'
+
+    if cellEdgeColours is not None:
+        if len(cellEdgeColours) != rows:
+            raise ValueError(
+                "'cellEdgeColours' must have {0} rows".format(rows))
```

Comment:
0 is unnecessary (and inconsistent with other messages). Though this could actually be written with an f-string:
```suggestion
                f"'cellEdgeColours' must have {rows} rows")
```